composer.json

```diff
@@ -16,7 +16,7 @@
         "php": "^5.5 || ^7.0",
         "ext-curl": "*",
         "php-http/httplug": "^1.0",
-        "php-http/message-factory": "^1.0",
+        "php-http/message-factory": "^1.0.2",
```

Comment:
We cant specify that the lowest version of `php-http/message-factory` is 1.0 when  rely on `php-http/message`, because `php-http/message` says that the lowest version is 1.0.2. 

https://travis-ci.org/php-http/curl-client/jobs/145511904